index.html

```diff
@@ -10,43 +10,66 @@
 <link rel="stylesheet" href="styles.css">
 <body>
     <header class="header">
-        <a href="#" class="nabar-brand"> My logo</a>
-    <nav class="navbar">        
-        <i class="fa fa-bars" class="menu-icon"></i>        
+        <a href="#" class="navbar-brand"> My logo</a>
+    <nav class="navbar">
+        <div class="hamburger">        
+            <i class="fa fa-bars" class="menu-icon"></i>
+        </div>
+        
+        <ul class="menu-list">
+            <li><a href="" class="menu">Portfolio</a></li>
+            <li><a href="" class="menu">About</a></li>
+            <li><a href="" class="menu">Contact</a></li>
```

Comment:
using a list is the best option for menus good job, I suggest using a # in your href, your browser will navigate to the appropriate parts of your page,